app/posts/manage-teacher-training-applications/2023-03-14-helping-users-decide-if-candidates-meet-the-itt-criteria.md

```diff
@@ -0,0 +1,98 @@
+
+We made this optional because by answering ‘No’ to any of the previous questions about the ITT criteria, the training provider would have already given some feedback to the candidate.
+
+![Screenshot showing a page with the heading 'Give Peter Essien any other feedback (optional)'. Beneath this is a set of 9 checkboxes labelled with different reasons. Beneath this is a green ‘Continue’ button.'](optional-feedback.png)
+
+If the training provider had answered ‘Yes’ to all of the questions about if the candidate met the ITT criteria, we show a page where the provider must give the candidate feedback on why they are rejecting their application.
+
+![Screenshot showing a page with the heading ‘Feedback for Peter Essien’. Beneath the heading is a exclaimation mark icon and the bolded text ‘You must not reject candidates for a lack of experience working in a school’. Beaneath this is the subheading ‘Tell Peter Essien why you are rejecting their application’. Beneath this is a set of 9 checkboxes labelled with different reasons. Beneath this is text box with the label ‘Give Peter Essien advice on what to do next. For example, “You should apply again to a different course”’. Beneath this is a green ‘Continue’ button.](feedback-page.png)
+
+## What we learnt
+
+We tested these designs with 9 training providers from different types of organisations. 6 people had admin roles and 2 had a lead role in their ITT team.
+
+When we tested the design we found that there were no major usability issues and providers were able to answer the questions because they would have assessed the candidate against the criteria before rejecting them.
+
+Most issues were around the ITT criteria itself, which our team does not have control over. We found that training providers:
+
+* hold the ITT criteria in high regard, and refer to it often when assessing candidates
+* feel there is a lot of ‘grey areas’ in the ITT criteria, especially when assessing “tricky” qualifications
+* find assessing international candidate’s qualifications the most time consuming – this may be leading to them rejecting these candidate prematurely
+* feel a third class degree is not strong enough for ITT – some providers mentioned that candidates need a 2:2 degree or higher to get a bursary from DfE which suggests that DfE thinks that 3rd class degrees are not the best
+* are not clear on the rules around if the degree subject should match the course subject
+* thought that the line “it’s the standard, not the certificate that matters” is too vague for them to use in practice
+* have more difficulty in assessing GCSEs against the criteria than degrees
+
+## Future considerations
+
+As a team, we do not have control over the ITT criteria. However, our user researcher and policy colleagues will play this research back to the team who look after the ITT criteria on GOV.UK, so that they can use this to make the guidance clearer.
```

Comment:
```suggestion
As a team, we do not have control over the ITT criteria. However, our user researcher and policy colleagues have discussed this research with the team who look after the ITT criteria on GOV.UK, so that they can use this to make the guidance clearer.
```